lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -280,18 +280,18 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
```

Comment:
I"m probably making it harder to merge my PR so this might be best left for later update, however we should we take the opportunity to improve fix it properly.

Assigning local vars for cert and key since they now get read several times from modules.params

Only checking if the file exists or trying to read the string if the values exist.  Otherwise rstrip will fail on a None.

cert = module.params.get('cert')
if cert:
  if os.path.isfile(cert):
    cert = open(cert, 'r').read().rstrip()
  else:
    cert = unicode(cert.rstrip())